# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp
--- a/media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp	Tue Sep 17 18:17:13 2019 +0000
+++ b/media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp	Tue Sep 17 18:17:14 2019 +0000
@@ -171,7 +171,8 @@
         if (mChild) {
           mChild->SendSetProxyConfig(dom::TabId(aProxyConfig.GetTabId()),
                                      aProxyConfig.GetLoadInfoArgs(),
-                                     aProxyConfig.GetAlpn());
+                                     aProxyConfig.GetAlpn(),
+                                     aProxyConfig.GetProxyPolicy());
         }
       },
       [](const nsCString& aError) {});